src/north/csvtohttp/CsvToHttp.class.js

```diff
@@ -46,6 +46,7 @@ class CsvToHttp extends ApiHandler {
 
     this.canHandleFiles = true
     this.canHandleValues = false
+    this.connected = true
```

Comment:
`this.connected = true` should it be instead set if the connect() has been successful in the parent class?